python/ql/src/semmle/python/web/twisted/Twisted.qll

```diff
@@ -36,15 +36,13 @@ predicate isTwistedRequestInstance(NameNode node) {
      */
 
     exists(Function func | func = node.getScope() |
-        func.getEnclosingScope() = aTwistedRequestHandlerClass().getScope()
-    ) and
-    (
-        /* Any parameter called `request` */
-        node.getId() = "request" and
-        node.isParameter()
-        or
-        /* Any request parameter of a known request handler method */
-        exists(Function func | node.getScope() = func |
+        func.getEnclosingScope() = aTwistedRequestHandlerClass().getScope() and
```

Comment:
Now that you've unified the two `exists` statements, you can move this conjunct alongside `func = node.getScope()` so that the final part of the `exists` only contains a disjunction. You then don't need the parentheses around the disjunction (as I'm sure autoformat will make clear).